timrlink.net.CLI.Test/GroupUsersExportDatabaseActionTest.cs

```diff
@@ -0,0 +1,307 @@
+
+            var options = InMemorySqLiteContextOptions();
+            GroupUsersDatabaseExportAction groupUsersAction;
+
+            {
+                var group1 = new API.Group
+                {
+                    description = "Series with airplanes for kids",
+                    externalId = "99C4",
+                    name = "Super Wings",
+                    parentExternalId = "77C1"
+                };
+                
+                var group2 = new API.Group
+                {
+                    description = "Nice animation series kids",
+                    externalId = "77C8",
+                    name = "Paw Patrol",
+                    parentExternalId = "77C1"
+                };
+                
+                var user1 = new User()
+                {
+                    uuid = "9d33c475-0da2-4b21-95b9-feca948cc80a"
+                };
+                
+                var user2 = new User()
+                {
+                    uuid = "80814e96-aa2b-4bbe-a7fe-ba25f3b60e2e"   
+                };
+                
+                var user3 = new User()
+                {
+                    uuid = "88a6ead0-4731-40e1-a5de-d873f094cace"
+                };
+                
+                var user4 = new User()
+                {
+                    uuid = "e25a59c8-d4de-4531-aae0-241a8c1c7115"
+                };
+                
+                var user5 = new User()
+                {
+                    uuid = "27da471e-9e72-4c3d-bc52-000c926d1ff9"
+                };
+                
+                var userGroup1 = new UserGroup()
+                {
+                    group = group1,
+                    users = new List<User> {user1, user2, user3}
+                };
+                
+                var userGroup2 = new UserGroup()
+                {
+                    group = group2,
+                    users = new List<User> {user3, user4, user5}
+                };
+
+                var memoryContext = new DatabaseContext(options);
+
+                var groupService = BuildGroupService(userGroup1, userGroup2);
+                groupUsersAction =
+                    new GroupUsersDatabaseExportAction(loggerFactory, memoryContext, groupService);
+            }
+
+            await groupUsersAction.Execute();
+
+            {
+                var memoryContext = new DatabaseContext(options);
+                var group1 = memoryContext.Group.Single(g => g.ExternalId == "99C4");
+                
+                Assert.AreEqual("Series with airplanes for kids", group1.Description);
+                Assert.AreEqual("Super Wings", group1.Name);
+                Assert.AreEqual("77C1", group1.ParentalExternalId);
+            }
+            
+            {
+                var memoryContext = new DatabaseContext(options);
+                var group1 = memoryContext.Group.Single(g => g.ExternalId == "77C8");
```

Comment:
group1 + group1

why are you doing this in extra scopes each time? 🤔

Reply:
This one is from database. The first one is `API.Group`. Do you think I should rename it?